src/traces/carpet/plot.js

```diff
@@ -96,13 +90,9 @@ function drawClipPath(trace, t, layer, xaxis, yaxis) {
     // This could be optimized ever so slightly to avoid no-op L segments
     // at the corners, but it's so negligible that I don't think it's worth
     // the extra complexity
-    trace.clipPathData = 'M' + segs.join('L') + 'Z';
-    clip.attr('id', trace.clipPathId);
-    path.attr('d', trace.clipPathData);
-        // .style('stroke-width', 20)
-        // .style('vector-effect', 'non-scaling-stroke')
-        // .style('stroke', 'black')
-        // .style('fill', 'rgba(0, 0, 0, 0.1)');
+    var clipPathData = 'M' + segs.join('L') + 'Z';
+    clip.attr('id', trace._clipPathId);
+    path.attr('d', clipPathData);
```

Comment:
Ah, thanks for removing. I left those because I so frequently wanted to toggle over to *displaying* the clip path instead of *using* the clip path. But no good reason to keep.